### DISCUSSION TITLE
Costum Theme Background Image not working, TailwindCSS

### DISCUSSION THREAD

**Discussion by jchaparr** [tailwindlabs/tailwindcss#9410] [Help] [answered]
I've customized a few background images in the Tailwind Config file. All the images load perfectly fine on my local machine but once I push it to github, the images no longer load. I'm not using any specific JS libraries or frameworks and I've made sure to re build before pushing my code.

Tailwind Config Code:
`backgroundImage: {
        'light-header-bg-mobile': "url('/images/bg-mobile-light.jpg')"`

HTML code:
` <header role="header-light" id="light-header" class="h-[200px] md:h-[300px] w-full bg-light-header-bg-mobile md:bg-light-header-bg-desktop"></header>`

Below are images from my local machine vs what github pages shows.

![Screen Shot 2022-09-24 at 1 48 14 PM](https://user-images.githubusercontent.com/81573972/192118320-ebbdf4c1-aaad-4a67-8bbe-d9d351e450b6.png)
![Screen Shot 2022-09-24 at 1 48 28 PM](https://user-images.githubusercontent.com/81573972/192118326-2f00d636-ef32-4eb4-bdda-5a8aadbdbc43.png)

**Comment by wongjn** [comment, marked as answer]
It seems like it could be a path problem. I see that [ToDo_FEM](https://github.com/jchapar/ToDo_FEM) seems to be your repository, and indeed it is visible at https://jchapar.github.io/ToDo_FEM/. I believe that when you are developing locally, you are probably looking at the site at a root host, like `http://localhost/`, in which case `/images/…` will resolve to `http:/localhost/images/…` which works fine. However on GitHub pages, `/images/…` resolves to `https://jchapar.github.io/images/…` but really it should resolve to `https://jchapar.github.io/ToDo_FEM/images/…`.

To resolve this path issue, you can set it to use a relative path. The path will be relative to the compiled CSS file, so in this case, to resolve to files in the images folder, use `../images`. For example, for the example you gave, try using `url('../images/bg-mobile-light.jpg')`.

**Reply by jchaparr** [reply]
Wow, I had tried using "./" and that didn't work. Didn't think to make the path be relative to my compiled CSS file. This worked. Thank you!
